src/ontology/cl-edit.owl

```diff
@@ -3524,17 +3524,16 @@ AnnotationAssertion(rdfs:comment obo:CL_0000046 "Consider using term 'pro-B cell
 AnnotationAssertion(rdfs:label obo:CL_0000046 "obsolete bursa processed stem cell"^^xsd:string)
 AnnotationAssertion(owl:deprecated obo:CL_0000046 "true"^^xsd:boolean)
 
-# Class: obo:CL_0000047 (neuronal stem cell)
+# Class: obo:CL_0000047 (neural stem cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "MESH:D058953"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:15247488"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://en.wikipedia.org/wiki/Neural_stem_cell"^^xsd:string) obo:IAO_0000115 obo:CL_0000047 "Neural stem cell is characterized as an undifferentiated cell that originates from the neuroectoderm and has the capacity both to perpetually self-renew without differentiating and to generate multiple types of lineage-restricted progenitors."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30639325"^^xsd:string) obo:IAO_0000115 obo:CL_0000047 "An undifferentiated neural cell that originates from the neuroectoderm and has the capacity both to perpetually self-renew without differentiating and to generate multiple central nervous system neuronal and glial cell types.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000047 "BTO:0002881"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000047 "CALOHA:TS-2360"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000047 "FMA:86684"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000047 "NSC"^^xsd:string)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000047 "neural stem cell"^^xsd:string)
-AnnotationAssertion(rdfs:label obo:CL_0000047 "neuronal stem cell"^^xsd:string)
-EquivalentClasses(obo:CL_0000047 ObjectIntersectionOf(obo:CL_0000048 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001017)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000047 obo:CL_0000048)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000047 "neuronal stem cell"^^xsd:string)
```

Comment:
Was uncertain if this term also included the "glial cell types" mentioned in the definition, but appears that it used that way as well: https://doi.org/10.1016/j.exger.2005.10.008. Consider adding as a dbxref.

Reply:
done!